Configs/vendor-documentation.php

```diff
@@ -1,6 +1,17 @@
 <?php
 
 return [
+    
+     /*
+    |--------------------------------------------------------------------------
+    | Locale
+    |--------------------------------------------------------------------------
+    |
+    | Language header docs
+    |
+    */
+    
+    'locale' => '.'.env('APIDOC_LOCALE'),
```

Comment:
Change the `env('APIDOC_LOCALE')` to `env('APIDOC_LOCALE', 'en')`  in case there is no `env` value is set.

Reply:
Done! f6b73ca8f9d7ddbcf55cab91da6f03662dcb0ae9